backend/app_fastapi_v2.py

```
@@ -217,6 +218,33 @@ async def get_user_details(current_user: dict = Depends(get_current_user)):
 
 
+# Set up your Stripe secret key
+stripe.api_key = Config.Stripe.SECRET_KEY
+
+class CheckoutSessionRequest(BaseModel):
+    price_id: str
+    success_url: str
+    cancel_url: str
+
+@v2.post("/stripe/create_checkout_session")
+async def create_checkout_session(request: CheckoutSessionRequest, current_user: dict = Depends(get_current_user)):
```

Ruff

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)